qemu/accel/tcg/cputlb.c

```diff
@@ -1555,6 +1556,12 @@ load_helper(CPUArchState *env, target_ulong addr, TCGMemOpIdx oi,
                     if (uc->stop_request)
                         break;
                 }
+#ifdef TARGET_ARM64
+                if (!handled) {
+                    addr = addr & ARM64_TBI_MASK;
+                    handled = 1;
+                }
+#endif
```

Comment:
I don't think this position is correct for this hack. Also it's inconsistent with the `store_helper`.